testutil/validatormock/component.go

```diff
@@ -20,6 +20,10 @@ import (
 // epochWindow is the window size of the lookahead.
 const epochWindow = 2
 
+// delayStartSlots is the number slots to wait before starting validatormock.
+// This is needed to avoid inconsistencies between peers on startup in simnet tests .
+const delayStartSlots = 2
```

Comment:
nit: combine the two `const` declarations in a single block

Reply:
done